src/app/lib/views/player/player.js

```diff
@@ -343,8 +343,6 @@
 
         onShow: function () {
             $('#header').removeClass('header-shadow').hide();
-            // Test to make sure we have title
-            win.info('Watching:', this.model.get('title'));
```

Comment:
that one i don't really care about, but the point of logging is that we put it there for some reason at some point.